src/configs/redux/store.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  Store,
+  combineReducers,
+  applyMiddleware,
+  createStore as originalCreateStore,
+} from 'redux';
+import { History } from 'history';
+import thunkMiddleware from 'redux-thunk';
+import { routerReducer } from 'react-router-redux';
+import { connectRouter, routerMiddleware } from 'connected-react-router';
+import application, { ApplicationState } from '../../reducers/application';
+
+export const createStore = (history: History): Store => {
+  const createStoreFunc = applyMiddleware(
+      thunkMiddleware,
+      routerMiddleware(history)
+    )(originalCreateStore),
+    allReducers = combineReducers({
+      routing: routerReducer,
+      router: connectRouter(history),
+      applicationState: application.reducer,
+    });
+
+  return createStoreFunc(allReducers, {
+    applicationState: ({
+      defaultData: 'DEFAULTING_FROM_STORE',
```

Comment:
not sure i understand what this is for.  can you help me understand what defaultData is?

Reply:
just dummy data to make sure i had the redux store setup correctly.  ill make sure it gets deleted